### PR TITLE
add alert message to m2k transform

### PR DESCRIPTION
**Pull request by RichardW98**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, use `fixes #<issue_number>(, fixes #<issue_number>, ...)` format, where issue_number might be a GitHub issue, or a Jira story (FLPATH-xxxx)*:
Fixes #

**Change type**
- [ ] New feature
- [ ] Bug fix
- [ ] Unit tests
- [ ] Integration tests
- [ ] CI
- [ ] Documentation
- [ ] Auto-generated SDK code

**Impacted services**
- [ ] Workflow Service
- [ ] Notification Service

**Checklist**
- [ ] Subject and description added to both, commit and PR.
- [ ] Relevant issues have been referenced.

### CODE REVIEW

File: workflow-examples/src/main/java/com/redhat/parodos/examples/move2kube/task/Move2KubeTransform.java [hunk 1]
```diff
@@ -73,27 +73,28 @@ public WorkReport execute(WorkContext workContext) {
...
 
-		String userId = String.valueOf(WorkContextUtils.getUserId(workContext));
-		if (!sendNotification(userId, workspaceId, projectId, transformId)) {
+		String message = sendNotification(workspaceId, projectId, transformId);
+		if (message == null) {
 			return new DefaultWorkReport(WorkStatus.FAILED, workContext,
 					new RuntimeException("Cannot notify user about the transformation status"));
 		}
-		return new DefaultWorkReport(WorkStatus.COMPLETED, workContext);
+		return new DefaultWorkReport(WorkStatus.COMPLETED, workContext, message);
```
Comment (gciavarrini) [inline workflow-examples/src/main/java/com/redhat/parodos/examples/move2kube/task/Move2KubeTransform.java:81]: but if it's `COMPLETED`, then why should we display an alert message?

**Comment by eloycoto** [comment]
TBH, the message should be get from checker here:
https://github.com/parodos-dev/parodos/blob/842b5748b84ac9cd880cb7968087b1f1d7f12fd6/workflow-service/src/main/java/com/redhat/parodos/workflow/execution/aspect/WorkFlowTaskExecutionAspect.java#L158-L160

File: workflow-examples/src/main/java/com/redhat/parodos/examples/move2kube/task/Move2KubeTransform.java (hunk 1 above)
Comment (RichardW98) [reply workflow-examples/src/main/java/com/redhat/parodos/examples/move2kube/task/Move2KubeTransform.java:81]: this is the task that checker will check. and checker wont' be exposed to UI

**Comment by RichardW98** [comment]
> workFlowTaskDefinition

yea..  so far now UI can't get checker status so we have to set the alert in task

**Review by masayag — APPROVED** [review]

**Pull request merged**